week3/src/components/funnel/FunnelTitle.tsx

```diff
@@ -0,0 +1,59 @@
+import { inputState } from '../../types/states';
+import styled from 'styled-components';
+
+interface FunnelTitleProps {
+  step: number;
+  input: inputState;
+}
+
+const FunnelTitle = ({ step, input }: FunnelTitleProps) => {
+  return (
+    <FunnelTitleWrapper>
+      <TitleWrapper>
+        {step === 0
+          ? '어떻게 추천해줄까?'
+          : step === 1
+          ? '먼저 나라 별로 골라볼까?'
+          : step === 2
+          ? '재료는 어떤게 좋아?'
+          : step === 3
+          ? '국물 있는거, 없는거?'
+          : '넌 오늘 이걸 먹어라'}
+      </TitleWrapper>
+      <StepWrapper>{input.recommendType === '취향' && step > 0 && step < 4 && step + ' / 3'}</StepWrapper>
```

Comment:
```suggestion
     {input.recommendType === '취향' && step > 0 && step < 4 && <StepWrapper>{`${step} / 3`}</StepWrapper>}
```
렌더링 자체를 결정할 수 있도록 이렇게는 어떤가용?